### PR TITLE
gzdoom: several improvements

### PR DESCRIPTION
**Pull request by mstraube**
- fix 'Cannot find gzdoom.pk3' with default config
 - install soundfont gzdoom.sf2
 - add .desktop file (from ubuntu package on zdoom.org)
 - set release date string

### CODE REVIEW

**Comment by Vaelatern** [comment]
Can we not have binary blobs in the repo? Can that image be found elsewhere?

**Comment by Vaelatern** [comment]
Same with that `.desktop` file, can we find it elsewhere?

**Comment by mstraube** [comment]
Both are in the .deb package provided by upstream. Would it be ok to add the .deb package to distfiles and extrat the files?

**Comment by Vaelatern** [comment]
That'd be better.

File: srcpkgs/gzdoom/template [hunk 1]
```diff
@@ -23,12 +25,24 @@ case "$XBPS_MACHINE" in
 	;;
 esac
 
+do_extract() {
+	tar --strip-components=1 -xf ${XBPS_SRCDISTDIR}/${pkgname}-${version}/g${version}.tar.gz
+	ar -p ${XBPS_SRCDISTDIR}/${pkgname}-${version}/${pkgname}_${version}_armhf.deb data.tar.xz |
+	tar --strip-components=4 -xJf - ./usr/share/{pixmaps/gzdoom.png,applications/gzdoom.desktop}
+}
```
Comment (Vaelatern) [inline srcpkgs/gzdoom/template]: Why not do the custom extraction in the `post_extract`?

**Comment by mstraube** [comment]
Hm, travis does not start... Do I have to do something else than updating my branch?

**Comment by Vaelatern** [comment]
Rebase on master to update the branch and push again?

**Pull request closed without merging**